src/main/java/org/protege/editor/owl/server/base/ProtegeServer.java

```diff
@@ -95,7 +93,7 @@ public void deleteUser(AuthToken token, UserId userId) throws AuthorizationExcep
             try {
                 User user = configuration.getUser(userId);
                 logger.info(printLog(token.getUser(), "Remove user", user.toString()));
-                manager.removeUser(user);
+                configuration = new ConfigurationBuilder(configuration).removeUser(user).createServerConfiguration();
```

Comment:
The removeXX() no longer throw these exception, but they are still thrown by the getUser(id) and getProject(id) methods.